base/math.jl

```diff
@@ -146,6 +146,10 @@ sqrt(x::Float32) = box(Float32,sqrt_llvm(unbox(Float32,x)))
 sqrt(x::Real) = sqrt(float(x))
 @vectorize_1arg Number sqrt
 
+# 1-arg hypot
+hypot(x::Real) = abs(promote(float(x))...)
```

Comment:
This is for the standard `hypot` function, that I didn't touch.  In the case of 3 arguments we don't know which is the minimum of the three args.